lib/gs-plugin-loader.c

```diff
@@ -3112,6 +3112,7 @@ gs_plugin_loader_generic_update (GsPluginLoader *plugin_loader,
 		helper->anything_ran = TRUE;
 		gs_plugin_status_update (plugin, NULL, GS_PLUGIN_STATUS_FINISHED);
 	}
+	gs_utils_update_install_timestamp (priv->settings);
```

Comment:
Nitpick: I'd have added a newline before the function call, otherwise it looks a bit cramped.